src/pages/project/dpad-logic.js

```diff
@@ -40,5 +40,18 @@ module.exports = {
         showRight: verifyCoordsExist(this.state.pages, getTargetCoords(this.state.zoomedPageCoords, 'right'))
       });
     }
+
+    if (this.state.params.mode === 'play' && this.state.isPageZoomed) {
+      // Pass bounding box of the currently focused page to the DPad for display logic
+      var elFocusedElementGroup = document.querySelector('.page-container:not(.blurred) .element-group');
+
+      if (elFocusedElementGroup) {
+        // There is a potential race condition here:
+        setTimeout(function () {
```

Comment:
I think the only way to improve this would be to use the expected size not the actual (like look at the window viewport, calculate the size of the page boundaries based on the scale and the height/width of a single page)